en/docs/install-and-setup/setup/performance-tuning/rabbitmq-transport-tuning.md

```diff
@@ -19,7 +19,6 @@ Sample configurations:
     parameter.password = "guest"
     parameter.retry_interval = "10s"
     parameter.retry_count = 5
-    parameter.connection_pool_size = 25
```

Comment:
In this case do we have to mention about the listner configs?

Reply:
Removed the listener configs